Sources/StreamChat/APIClient/Endpoints/PollsEndpoints.swift

```diff
@@ -0,0 +1,170 @@
+        .init(
+            path: .polls,
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: createPollRequest
+        )
+    }
+    
+    static func updatePoll(updatePollRequest: UpdatePollRequestBody) -> Endpoint<PollPayload> {
+        .init(
+            path: .polls,
+            method: .put,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: updatePollRequest
+        )
+    }
+    
+    static func queryPolls(queryPollsRequest: QueryPollsRequestBody) -> Endpoint<QueryPollsPayload> {
+        .init(
+            path: .pollsQuery,
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: queryPollsRequest
+        )
+    }
+    
+    static func deletePoll(pollId: String) -> Endpoint<EmptyResponse> {
+        .init(
+            path: .poll(pollId: pollId),
+            method: .delete,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: nil
+        )
+    }
+    
+    static func getPoll(pollId: String) -> Endpoint<PollPayload> {
+        .init(
+            path: .poll(pollId: pollId),
+            method: .get,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: nil
+        )
+    }
+    
+    static func updatePollPartial(
+        pollId: String,
+        updatePollPartialRequest: UpdatePollPartialRequestBody
+    ) -> Endpoint<PollPayload> {
+        .init(
+            path: .poll(pollId: pollId),
+            method: .patch,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: updatePollPartialRequest
+        )
+    }
+    
+    static func createPollOption(
+        pollId: String,
+        createPollOptionRequest: CreatePollOptionRequestBody
+    ) -> Endpoint<PollOptionResponse> {
+        .init(
+            path: .pollOptions(pollId: pollId),
+            method: .post,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: createPollOptionRequest
+        )
+    }
+    
+    static func updatePollOption(
+        pollId: String,
+        updatePollOptionRequest: UpdatePollOptionRequest
+    ) -> Endpoint<PollOptionResponse> {
+        .init(
+            path: .pollOptions(pollId: pollId),
+            method: .put,
+            queryItems: nil,
+            requiresConnectionId: false,
+            requiresToken: true,
+            body: updatePollOptionRequest
+        )
+    }
+    
+    static func deletePollOption(pollId: String, optionId: String) -> Endpoint<EmptyResponse> {
```

Comment:
You can see that here we have the "Response" post-fix

Reply:
fixed all these, I was following the backend conventions the first time